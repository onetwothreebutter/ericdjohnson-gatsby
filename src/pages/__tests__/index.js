import IndexPage from '../index'

test('It workss', () => {
    const x = '';
    expect(2).toBe(2);
})