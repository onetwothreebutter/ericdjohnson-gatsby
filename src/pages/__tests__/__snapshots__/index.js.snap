// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page should render correctly 1`] = `
<div>
  <div
    className="homepage-background eric-and-elwood"
  >
    <div
      className="just-eric"
    />
    <div
      className="just-eric"
    />
    <div
      className="just-eric"
    />
    <div
      className="just-eric"
    />
    <div
      className="just-eric"
    />
  </div>
  <div
    className="homepage-background-helper"
  />
  <div
    className="easter-egg"
  />
  <div
    className="easter-egg__video-container"
  >
    <div
      className="easter-egg__video"
      id="easter-egg__video"
    />
    <div
      className="easter-egg__close"
    >
      âœ•
    </div>
  </div>
  <img
    className="easter-egg__appendage"
    src=""
  />
  <div
    className="homepage-container"
  >
    <div
      className="homepage-scaling-container"
    >
      <section
        className="homepage-banner"
      >
        <h1
          className="homepage-banner__heading"
        >
          Eric Johnson
        </h1>
        <div
          className="homepage-banner__subheading"
        >
          Web Developer, UX Guy, 
          <br />
           Llama Enthusiast
        </div>
      </section>
      <Menu />
    </div>
  </div>
  <div
    className="photo-focus"
  >
    <button
      className="photo-focus__button -focus -active"
    >
      focus photo
    </button>
    <button
      className="photo-focus__button -blur"
    >
      blur photo
    </button>
  </div>
</div>
`;
